File: backend-java/src/main/java/mx/codemx/modules/evaluacion/TipoEvaluacion.java
package mx.codemx.modules.evaluacion;

/** Estrategias de evaluación disponibles (entrada del Factory Method). */
public enum TipoEvaluacion {
    /** Ejecuta el código en el Servicio Python externo (HTTP). */
    REMOTA,
    /** Ejecuta el código localmente con un proceso python3 (respaldo). */
    LOCAL
}

File: backend/Modules/Evaluacion/TipoEvaluacion.cs
namespace CodeMX.Api.Modules.Evaluacion;

/// <summary>Estrategias de evaluación disponibles (entrada del Factory Method).</summary>
public enum TipoEvaluacion
{
    Remota,  // ejecuta el código en el Servicio Python externo (HTTP)
    Local    // ejecuta el código localmente con un proceso python3 (respaldo)
}
